frontend/src/stores/assignment.ts

```diff
@@ -83,6 +86,59 @@ export const useAssignmentStore = defineStore('assignment', () => {
     });
   }
 
+  function getAssignmentDataAPI(): IAssignmentAPI {
+    const formattedContacts: IAssignmentInfoDataAPI[] = [];
+
+    assignment.contacts.forEach(contact => {
+      const formattedPhoneNumbers: IAssignmentInfoDataAPI[] = [];
+      const formattedAddresses: IAssignmentInfoDataAPI[] = [];
+
+      contact.phoneNumbers.forEach(phoneNumber => {
+        formattedPhoneNumbers.push({
+          type: phoneNumber.type,
+          number: phoneNumber.number
+        })
+      })
+
+      contact.addresses.forEach(address => {

```

Comment:
it's better to use .map here it will make the values immutable and code more robust

e.g. const formattedPhoneNumbers = contact.addresses.map(address => {})

Reply:
Done